eyewear_shop/demo/loyalty_program.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="loyalty_program_2" model="loyalty.program">
+        <field name="name">10% Discount for products worth 1000$</field>
+        <field name="program_type">coupons</field>
+        <field name="portal_point_name">Coupon point(s)</field>
+        <field name="trigger">with_code</field>
+        <field name="company_id" ref="base.main_company"/>
+        <field name="mail_template_id" ref="loyalty.mail_template_loyalty_card"/>
+    </record>
+    <record id="loyalty_program_3" model="loyalty.program">
+        <field name="name">10% on Rayban</field>
+        <field name="total_order_count">1</field>
+        <field name="trigger">auto</field>
+        <field name="portal_point_name">Promo point(s)</field>
+        <field name="company_id" ref="base.main_company"/>
+    </record>
+    <record id="loyalty_program_4" model="loyalty.program">
+        <field name="name">Buy2Get1 Carrera</field>
+        <field name="program_type">buy_x_get_y</field>
+        <field name="date_to">2023-08-15</field>
+        <field name="trigger">auto</field>
+        <field name="portal_point_name">Credit(s)</field>
+        <field name="company_id" ref="base.main_company"/>
+    </record>
+    <record id="loyalty_program_5" model="loyalty.program">
+        <field name="name">15% Festival Discount</field>
+        <field name="program_type">promo_code</field>
+        <field name="total_order_count">4</field>
+        <field name="trigger">with_code</field>
+        <field name="portal_point_name">Discount point(s)</field>
+        <field name="company_id" ref="base.main_company"/>
+    </record>
+    <record id="loyalty.gift_card_program" model="loyalty.program">
+        <field name="name">Gift Cards</field>
+        <field name="is_payment_program" eval="True"/>
+        <field name="applies_on">future</field>
+        <field name="program_type">gift_card</field>
+        <field name="portal_visible" eval="True"/>
+        <field name="trigger">auto</field>
+        <field name="portal_point_name">$</field>
+        <field name="company_id" ref="base.main_company"/>
+        <field name="trigger_product_ids" eval="[(6, 0, [ref('loyalty.gift_card_product_50')])]"/>
+        <field name="mail_template_id" ref="loyalty.mail_template_gift_card"/>
+    </record>
```

Comment:
This is a standard data, please only apply what is new from the industry (and I don't see any here)
```suggestion
```